requirements.txt

```diff
@@ -31,7 +30,6 @@ python-dateutil==2.6.1
 python-gnupg==0.4.1
 pytz==2017.3
 bleach==2.1.3
-html5lib==0.999999999
```

Comment:
I believe that we pinned to this specific version of `html5lib` because it was required by `bleach`.  It looks like[ Bleach v 3.0.0 no longer depends on html5lib](https://pypi.org/project/bleach/) but we would need to upgrade bleach.

Reply:
That makes sense to pin to a version so that, if we're also using it, it won't conflict, but it should be, now that we don't conflict, it will use the version that's in its requirements, yes?

Reply:
I'm a little confused by what you're saying ^.  I have a vague memory of needing to pin to a specific version of html5lib to get bleach to work properly back in the day.  I'm not saying that is still the case today, but I have enough concern about it to not want to merge directly into develop without proving this.

https://github.com/pattisdr/osf.io/commit/8ade5bf23206657d577d8082913ea62bd6c4511f#diff-b4ef698db8ca845e5845c4618278f29a

PR upgrading bleach, might be useful in seeing locations where bleach could be tested.
https://github.com/CenterForOpenScience/osf.io/pull/7962/